README.md

```diff
@@ -12,42 +12,40 @@ management, role based access control etc to first and third-party Software as a
 Azure AD Client Connector allows you to perform operations on users and groups of an Azure AD. The operations are 
```

Comment:
Shall we rephrase the above paragraph as: "Azure Active Directory(AD) is Microsoft’s multi-tenant, cloud-based directory and identity management service. It provides capabilities to enable Single Sign-On (SSO), multi-factor authentication, self-service password reset, privileged identity management, role-based access control, etc., to first and third-party Software as a Service (SaaS) applications.	"

Reply:
What is Azure AD Client Connector? Is it a term Azure uses?

Reply:
Let's rephrase this as: "Azure AD Client Connector allows you to perform operations on users and groups in an Azure AD using the v1.0 version of Microsoft Graph API. The client supports OAuth 2.0 authentication."

Reply:
The Client part is unnecessary. Added the suggestion